2021-11-03 09:07:29,868 INFO    MainThread:103501 [wandb_setup.py:_flush():69] setting env: {}
2021-11-03 09:07:29,868 INFO    MainThread:103501 [wandb_setup.py:_flush():69] setting login settings: {}
2021-11-03 09:07:29,869 INFO    MainThread:103501 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211103_090729-1cmhtbhh/logs/debug.log
2021-11-03 09:07:29,869 INFO    MainThread:103501 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211103_090729-1cmhtbhh/logs/debug-internal.log
2021-11-03 09:07:29,869 INFO    MainThread:103501 [wandb_init.py:init():381] calling init triggers
2021-11-03 09:07:29,869 INFO    MainThread:103501 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-11-03 09:07:29,869 INFO    MainThread:103501 [wandb_init.py:init():430] starting backend
2021-11-03 09:07:29,869 INFO    MainThread:103501 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-03 09:07:29,937 INFO    MainThread:103501 [backend.py:ensure_launched():135] starting backend process...
2021-11-03 09:07:30,022 INFO    MainThread:103501 [backend.py:ensure_launched():140] started backend process with pid: 104894
2021-11-03 09:07:30,025 INFO    MainThread:103501 [wandb_init.py:init():435] backend started and connected
2021-11-03 09:07:30,029 INFO    MainThread:103501 [wandb_init.py:init():494] updated telemetry
2021-11-03 09:07:30,030 INFO    MainThread:103501 [wandb_init.py:init():517] communicating current version
2021-11-03 09:07:30,960 INFO    MainThread:103501 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.6 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-11-03 09:07:30,961 INFO    MainThread:103501 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-11-03 09:07:31,112 INFO    MainThread:103501 [wandb_init.py:init():557] starting run threads in backend
2021-11-03 09:07:36,121 INFO    MainThread:103501 [wandb_run.py:_console_start():1605] atexit reg
2021-11-03 09:07:36,121 INFO    MainThread:103501 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-11-03 09:07:36,122 INFO    MainThread:103501 [wandb_run.py:_redirect():1484] Redirecting console.
2021-11-03 09:07:36,124 INFO    MainThread:103501 [wandb_run.py:_redirect():1540] Redirects installed.
2021-11-03 09:07:36,125 INFO    MainThread:103501 [wandb_init.py:init():582] run started, returning control to user process
2021-11-03 09:08:30,544 INFO    MainThread:103501 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 1
2021-11-03 09:08:30,545 INFO    MainThread:103501 [wandb_run.py:_restore():1547] restore
2021-11-03 09:08:33,538 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 09:08:34,049 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 09:08:34,178 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-11-03 09:08:35,194 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 22352
}

2021-11-03 09:08:35,298 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 22352
}

2021-11-03 09:08:35,401 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:35,503 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:35,606 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:35,709 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:35,819 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:36,001 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}

2021-11-03 09:08:36,272 INFO    MainThread:103501 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 22352
  total_bytes: 22352
}
local_info {
}

2021-11-03 09:08:37,657 INFO    MainThread:103501 [wandb_run.py:_show_files():1947] logging synced files
